### ISSUE TITLE
iOS build bug

### ISSUE THREAD

**Issue by seyedmmousavi** [wang-bin/QtAV#707]
Hello dear my friend,
After successful building ffmpeg as Universal for iOS (Using [https://github.com/wang-bin/build_ffmpeg](https://github.com/wang-bin/build_ffmpeg), I built the QtAV as Release without examples.
After executing `sdk_install.sh` (and two copy command error-Issue #650) I started to building QMLPlayer. It failed with following error message: 
```
ld: framework not found Cocoa
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
Steps for reproducing error:

1. Create a project or open an example project in QtCreator
2. Run project in iPhone Simulator. It will run without problem.
3. Open `main.qml` file and add `import QtAV 1.7` in first line.
4. Run project again and it will fails with following error:
```
ld: framework not found Cocoa
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
5. Open `<Projectname>.xcodeproj` file from the build folder
6. Run project as Release
7. You will see error `module "QtAV" plugin "QmlAV" not found`

Can you help me please?

**Comment by kolegs** [comment]
Got the same problem with real device(iPhone). 
Checked and examples also generate the same problem.

Also if you open <projectName>.xcodeproj file and remove Cocoa.framework the program will compile but QML plugins will not work.

**Comment by seyedmmousavi** [comment]
**I found a solution**
Edit each *.prl files from /path/to/Qt/version/ios/qml/QtAV/ and remove "-framework Cocoa " option from the last line.

**Comment by kolegs** [comment]
You can also before building QtAV change file libQtAV.pro(line 420) and simply remove -framework Cocoa.

But still after installing the plugin when I add ```import QtAV 1.6``` my QML file will not load

**Comment by seyedmmousavi** [comment]
Yes it works for me. I used head of master branch of the QtAV and Qt 5.6.1 and Xcode 7.3.1 andffmpeg 3.1.2.
Also I use `import QtAV 1.7` not 1.6
Since I'm busy now, I will recheck my process steps again within next 3 days.

**Comment by seyedmmousavi** [comment]
In isuue #650 > step 3 > 2 last step updated

> goto build folder and copy /bin/QtAV folder to /path/to/Qt/version/ios/qml/QtAV folder
changed to:
> goto build folder and copy /bin/QtAV/* to /path/to/Qt/version/ios/qml/QtAV/ folder

I successfully tested procedure (in Issue #650) for Qt 5.7 and it works.

**Comment by redfearnk** [comment]
QtAV does not work for Qt 5.8
